### PR TITLE
LPS-120620 set container id since it is needed to render the form

### PR DESCRIPTION
**Pull request by marcosapmf**
Related issue: https://issues.liferay.com/browse/LPS-120620

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_general.scss [hunk 1]
```diff
@@ -158,6 +158,19 @@ td.lfr-name-column {
 	}
 }
 
+.editing-form-entry-admin {
+	background-color: white;
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_general.scss:162]: @marcosapmf, how about add a line break after this line?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_general.scss [hunk 2]
```diff
@@ -158,6 +158,19 @@ td.lfr-name-column {
...
+	.ddm-form-basic-info {
+		padding-bottom: 8px;
+		padding-left: 20px;
+		padding-top: 15px;
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_general.scss:166]: @marcosapmf, how about add a line break after this line?

**Comment by natocesarrego** [comment]
ci:forward

**Comment by natocesarrego** [comment]
ci:forward

**Comment by natocesarrego** [comment]
@marcosapmf, can you please rebase and resend from your repo? I'm sure the reported issue isn't related to your pull.

Thanks.

**Pull request closed without merging**